### PR TITLE
Cycle gamma scalper on timer & orderIds

### PR DESCRIPTION
**Pull request by DonDuala**

### CODE REVIEW

File: src/scalper.ts [hunk 1]
```diff
@@ -181,13 +195,31 @@ async function scalperPerp() {
...
+  console.log('Ask', gammaAsk, 'ID', gammaAskID)
+
+  // Check by periods per scalperWindow for fills matching either gamma scalp and rerun after scalperWindow expires
+  let periods = 180;
+  let timeWaited = 0;
+  let filledBidGamma: number;
+  let filledAskGamma: number;
+  while (timeWaited < scalperWindow){
+    // Check this was buggy here updating account orders
+    // mangoAccount = (await client.getMangoAccountsForOwner(mangoGroup, owner.publicKey))[0];
```
Comment (brittcyr) [inline src/scalper.ts:210]: remove unneeded comments like this

**Review by brittcyr — COMMENTED** [review]
Can you put in a big comment at the top of the main function that describes what it is expecting as inputs from the risk manager and any state invariants it tries to maintain.

File: src/scalper.ts (hunk 1 above)
Comment (DonDuala) [reply src/scalper.ts]: Will remove later, for now just for benefit of the Mango team

Comment (DonDuala) [reply src/scalper.ts]: // TODO Risk Manager feeds in new DIP position automatically
// Risk Manager sends new DIP positions upon User deposit into DIPs without MM fill
// See DIP object below for fields

Sufficient?

**Comment by DonDuala** [comment]
Merging earlier for Mango's review

**Pull request merged**